### DISCUSSION TITLE
Outreach

### DISCUSSION THREAD

**Discussion by mrchristian** [petermr/semanticClimate#31] [General]
Make preperations for the FSCI 31 July to 4th Aug event - which is a hackathon and keynote. See: https://force11.org/fsci/post/plenary-events-2023/

**Comment by mrchristian** [comment]
First off is to get SC organisers signed up on FSCI systems: Thats @petermr @ShweataNHegde and Gitanjali Yadav.

Please signup here: https://docs.google.com/forms/d/e/1FAIpQLSffHOw7zKUVFZfRwtLkTNrq5bloRMxtMQm6J5ISXD7sLptQtg/viewform

Select course E08 as your course (this is another course I'm running) but until they have the hackathon as acourse in their systems please use this one.

Signup status:

- [ ] PMR - TBC
- [ ] Shweata - TBC 
- [ ] Gitaanjali - TBC


To do list for signup and Hackathon visibility with FSCI

- [ ] Have FSCI create a project for the Hackathon, or we create one on the platform they use OSF - like this https://osf.io/t4j5a/

**Comment by mrchristian** [comment]
There is a second part to signup for Eventbrite and Sched. I have fowarded you the details. Sorry about this is a bit complicated and fragmented.

Eventbrit and Sched sign up status

- [ ] Simon
- [ ] PMR
- [ ] Shweata
- [ ] Gitaanjali

**Comment by petermr** [comment]
Simon,
I am now lost,
I have filled in "the same form" 4 times. I have no idea whether I have
registered 0, 1 or 4 times.

On the plus side Shweata was very much welcomed today.

On Fri, Jun 23, 2023 at 3:40 PM Simon Worthington ***@***.***>
wrote:

> First off is to get SC organisers signed up on FSCI systems: Thats
> @petermr <https://github.com/petermr> @ShweataNHegde
> <https://github.com/ShweataNHegde> and Gitanjali Yadav.
>
> Please signup here:
> https://docs.google.com/forms/d/e/1FAIpQLSffHOw7zKUVFZfRwtLkTNrq5bloRMxtMQm6J5ISXD7sLptQtg/viewform
>
> Select course E08 as your course (this is another course I'm running) but
> until they have the hackathon as acourse in their systems please use this
> one.
>
> Signup status:
>
>    - PMR - TBC
>    - Shweata - TBC
>    - Gitaanjali - TBC
>
> To do list for signup and Hackathon visibility with FSCI
>
>    - Have FSCI create a project for the Hackathon, or we create one on
>    the platform they use OSF - like this https://osf.io/t4j5a/
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/petermr/semanticClimate/discussions/31#discussioncomment-6263018>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFTCS6WCJKDBSW5BLRFQ5TXMWTHNANCNFSM6AAAAAAZRUXBUU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Peter Murray-Rust
Founder ContentMine.org
and
Reader Emeritus in Molecular Informatics
Dept. Of Chemistry, University of Cambridge, CB2 1EW, UK

**Comment by petermr** [comment]
# core activities for delegates in FSCI hack

# core themes

* climate disaster gets worse every month. WE have to inform OURselves - no-one is going to do it for us
* young people are critical. We specifically cater for 15-year old citizens and non-academics
* base language is EN, but semantic structure and Wikidata makes interpretation, linking and translation much easier.
* we build a ClimateExplorer and LearnByExploring. 
* emphasis is on text (links to images and data are kept, but not emphasized)
* we are extremely faithful to the text in the reports, but make it easier to understand and navigate, 

# exploration

We choose 3 axes along which to explore, Delegates could explore 
## chapters
there are about 70 chapters, all the size of a book. We convert to HTML, convert to easier reading (e.g. separate chunks with hyperlinks). We offer 3 pre-prepared chapters , each with an expert guide:
  * Shiwani: SROCC (Oceans and **_Cryosphere_**). Chapter02 (**High Mountains**)
  * Yasin: SRCCL (**_Changing Climate Landscape_**), Chapter05 (**Food Security**) and WG3/Chapter07 (Agriculture)
  * CoreTeam/PMR WG3 (Urban) Chapter08 (**Cities**)
  
Alternatively if you have a special interest (e.g. the Ocean) let us know several days beforehand.

## statements

The Synthesis Report SYR/ LongerReport has about 700+ statements about climate (causes, current state, predictions). These are very powerful but are presented in running text. We split them into semantic sentences, label them and store in a Wikibase which can be queried with SPARQL (the language for knowledge graphs).

## glossaries
Each of the 6 reports (WG1/2/3 , SROCC., SR15, SRCCL) have an extensive glossary with implicit linking. We turn the links into HTML to create a Knowledgebase for Climate which can be used for annotating the chapters or other documents. Also we index the terms in Wikidata which gives a rich background knowledgebase.



delegates can choose one of these. Depending on numbers each chapter may have a team of 3-4 people. Questions to answer include:
* do I/we find this useful?
* what does it not cover?
* what was hard to understand?
* did the #semanticClimate structure/annotation make it easier to read and understand the IPCC materials?

# clientele

We have been approached by many people who want to know the definitive state of the earth's climate. They are prepared to read the IPCC material but need help with navigation , and understanding terms. They include :
* individual (non-climate) scientists
* teenagers
* citizen groups (e.g. a book-reading club)
* local government officers
* other climate action groups


# background

* We expect 15-20 participants,
* anticipate ca 50% associated with academic libraries in some way
* anticipate 75% non-tech (but will be able to use tools)
*  mainly asynchronously but with 1-hour sign-ins/signoffs each day (note timezone problems. 
* Runs from 2023-07-31 (Mon) - 2023-08-04 (Friday). 

# questions 

* frameworks for delivery, storage and communication?
* central servers? 
* working in teams
* breakout rooms?

**Comment by mrchristian** [comment]
City Climate Plans: Reader Collation Prototype

Description: The objective is if we can see if we have a way forward to make a Proof of Concept demo for 'City Climate Plan' readers from the IPCC reports. The idea being that pyami can already collect filtered lists of texts from IPCC reports - and to see if there is a way to post-process this content using the Quarto framework? 

Please add any notes or comments

**Comment by mrchristian** [comment]
Here is a draft for review of a pubic description of the 'IPCC Report and City Climate Change Plans' which I'll make into a new discussion thread - https://docs.google.com/document/d/1uhd1bhtsfYTjZ6g3YQTw6nvECQ_RwezBt_svlKFP5FE/edit#heading=h.1h3q4uir3ua2

**Comment by petermr** [comment]
Thanks Simon, this is great.

The team (Mahvish, Renu, Waheb, PMR) is going to write:
* an overview of the IPCC corpus (the seven Reports)
* an overview of the detailed structure of a report
* the conversion of the Cities chapter (WG3/Ch08)

P.


On Thu, Jul 6, 2023 at 3:51 PM Simon Worthington ***@***.***>
wrote:

> Here is a draft for review of a pubic description of the 'IPCC Report and
> City Climate Change Plans' which I'll make into a new discussion thread -
> https://docs.google.com/document/d/1uhd1bhtsfYTjZ6g3YQTw6nvECQ_RwezBt_svlKFP5FE/edit#heading=h.1h3q4uir3ua2
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/petermr/semanticClimate/discussions/31#discussioncomment-6375208>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFTCSZLU4A4UMIQSIJNQCDXO3GFNANCNFSM6AAAAAAZRUXBUU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Peter Murray-Rust
Founder ContentMine.org
and
Reader Emeritus in Molecular Informatics
Dept. Of Chemistry, University of Cambridge, CB2 1EW, UK
